styles.css

```diff
@@ -0,0 +1,79 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  width: 100%;
+  max-width: 375px;
+  font-family: 'poppins', sans-serif;
+}
+
+.background-img {
+  background-image: url(./Images/header-bg.png);
+  background-repeat: no-repeat;
+  height: 680px;
+}
+
+.header-flex {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  align-items: center;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.logo-link {
+  font-weight: 700;
+  font-size: 18px;
+  color: #6070ff;
+  padding: 10px 0 0 10px;
+}
+
+.union-icon {
+  width: 18px;
+  height: 12px;
+  margin-right: 10px;
+  cursor: pointer;
+}
+
+.heading {
+  font-weight: 700;
+  font-size: 40px;
+  max-width: 327px;
+  height: 150px;
+  line-height: 52px;
+  color: #172b4d;
+  margin: 80px 0 0 24px;
+}
+
+.self-intro {
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+  max-width: 327px;
+  margin: 18px 0 0 24px;
+}
+
```

Comment:
- [ **OPTIONAL** ] **IMPORTANT && RECOMMENDED** I think it would be better if you remove all the fixed widths(e.g: max-width: 375px, 327px,...) in your code base to make your page be responsive in every screen size. If you use those fixed sizes it will be difficult for you when you will start the **Desktop version** in the next projects.

Reply:
@tresorsawasawa Thank you so much sir for your recommendation sir. But I want to make it specifically for mobile phones that is why I did fixed widths. But promise to make it responsive in the upcoming week. This way, it is more easy to build it. 
Thank you.